### ISSUE TITLE
On adding component "There is more than one file for Spanish language, please adjust the file mask and use components for translating different resources."

### ISSUE THREAD

**Issue by eighthave** [WeblateOrg/weblate#9214]
### Describe the issue

I'm adding a new component to F-Droid using very simple YAML files.  It uses a typical monolingual base file and subdirs for each language.  This is the same file layout as https://hosted.weblate.org/projects/f-droid/website uses for JSON files and https://hosted.weblate.org/projects/f-droid/anti-features uses for YAML files:

* File mask: `badge/src/*/strings.yml`
* Monolingual base language file: `badge/src/strings.yml`

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar issues in this repository.

### Steps to reproduce the behavior

https://gitlab.com/fdroid/artwork.git

It has these files that might be relevant:
```console
$ ls -1 badge/src/es*/*
badge/src/es-419/strings.yml
badge/src/es-ar/strings.yml
badge/src/es-es/strings.yml
badge/src/es-mx/strings.yml
badge/src/es/strings.yml
```

This is perhaps related to the `SIMPLIFY_LANGUAGES` setting.  If so, perhaps the error code can mention that?

### Expected behavior

This should be allowed.

### Screenshots

![Screenshot from 2023-05-09 11-07-03](https://user-images.githubusercontent.com/143465/237049266-8d9233f2-28bf-436f-88d5-bc9ca5c799ea.png)

### How do you run Weblate?

weblate.org service

**Comment by eighthave** [comment]
Showing the filenames in question sounds like a good solution to me.

**Comment by nijel** [comment]
I have no clue why GitHub didn't link this together. Fixed by 37e61f4b22ebcbeb8cb3483b54f3419934aa668a
